package.json

```diff
@@ -112,14 +112,15 @@
   },
   "dependencies": {
     "@gooddata/gooddata-js": "^11.20.0",
-    "@gooddata/goodstrap": "^64.3.0",
+    "@gooddata/goodstrap": "^64.6.0",
     "@gooddata/js-utils": "^3.5.0",
     "@gooddata/numberjs": "^3.2.3",
     "@gooddata/typings": "^2.19.0",
     "ag-grid-community": "20.0.0",
     "ag-grid-react": "20.0.0",
     "classnames": "^2.2.6",
     "custom-event": "^1.0.1",
+    "date-fns": "^2.7.0",
```

Comment:
Adding date-fns seemed better than to use `moment` that is transitively already in the bundle (because of goodstrap), because it is much easier to use and can be tree shaken (as opposed to moment), so we only include the one function we need. I'm willing to change this to moment if the consensus is leaned towards it :)